### PR TITLE
Election create event

### PR DESCRIPTION
**Pull request by tank157**
Reopening this against master, incorporated a few tweaks from Eric's final rounds.

I also changed the use of `:ocd-division-id` on the user address data to `:ocd-divisions` to match the work over [here](https://github.com/democracyworks/user-works/commit/2a3a9b6815968a0ecff4ab1c336bb19cf2d287f8).

### CODE REVIEW

**Comment by tank157** [comment]
Giving back to Eric for one more look

File: test/electorate_works/handlers_test.clj [hunk 1]
```diff
@@ -132,24 +132,33 @@
 
 (def user-with-one-ocd
   {:id (java.util.UUID/randomUUID)
-   :addresses {:registered {:ocd-division-id #{"ocd-division/country:us"}}}})
+   :addresses {:registered {:ocd-divisions #{"ocd-division/country:us"}}}})
 
 (defn ref-one-election [& args]
   (future
     {:elections #{{:id (java.util.UUID/randomUUID)
                    :district-divisions #{"ocd-division/country:us"}}}}))
```
Comment (ericnormand) [inline test/electorate_works/handlers_test.clj]: Do we have a card to change `:district-divisions` to `:ocd-divisions` in election-works?

**Comment by ericnormand** [comment]
LGTM

File: test/electorate_works/handlers_test.clj (hunk 1 above)
Comment (tank157) [inline test/electorate_works/handlers_test.clj]: I don't think we do. That particular naming convention does come from the election data model doc, if I recall correctly, but if we've decided to change it in the meantime, then yeah, let's get a card for it. @cap10morgan, are we retaining the name `:district-divisions` on the Election type?

Comment (turbovoteops) [inline test/electorate_works/handlers_test.clj]: "district-divisions" was intentionally chosen on the election side. I could see changing it to "district-ocd-divisions", though.

The "district" part is giving context to the attribute. It's saying, "These are the OCD division IDs that comprise this election's district."

In the other uses, the context seemed to already be there (such as when they are a component of an address or a part of an election authority).

But if you guys disagree and would like to propose improvements, feel free. Naming things, sheesh!



- Wes

**Pull request merged**